app/views/games/show.html.erb

```diff
@@ -8,11 +9,12 @@
   </div>
 </div>
 <table class="chessboard">
-  <% (0..7).each do |row| %>
+  <% (0..7).to_a.reverse.each do |row| %>
     <tr class="x-position-<%= row %>" >
       <% (0..7).each do |column| %>
-        <td class="y-position-<%= column %>" x-position-data="<%= row %>" y-position-data="<%= column %>" ></td>
+        <%= place_piece_td(row, column).html_safe %>
```

Comment:
used 'html_safe' since all the characters get escaped if you don't use it. (don't want to use this method with user-input)